### PR TITLE
refactor(version,kubernetes): Bump kubernetes version from 1.13.0 to 1.14.0

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This commit bumps kubernetes dependency's version from 1.13.0 to 1.14.0 and also adds a util package for byte conversion as these codes are removed from kubernetes 1.14.0 codebase.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
refactor(version,kubernetes): Bump kubernetes version from 1.13.0 to 1.14.0
```

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+func BytesToGiB(volumeSizeBytes int64) int64 {
+	return volumeSizeBytes / GiB
+}
+
+// GiBToBytes converts GiB to Bytes
+func GiBToBytes(volumeSizeGiB int64) int64 {
+	return volumeSizeGiB * GiB
+}
+
+func ParseEndpoint(endpoint string) (string, string, error) {
```
Comment (andyzhangx) [inline pkg/util/util.go]: do we need this function? and where do the other functions come from?

Comment (ashishranjan738) [reply pkg/util/util.go]: we don't removing it.

Comment (ashishranjan738) [reply pkg/util/util.go]: I took this file from here https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/util/util.go

Comment (ashishranjan738) [reply pkg/util/util.go]: removed

**Pull request merged**